zh_CN.ts

```diff
@@ -276,8 +276,8 @@ export const ZH = {
     OfflineEarningTimeDesc: "您能够获取离线收益的时长",
     BuyPermit: "购买许可证",
     NotEnoughBuildingPermit: "没有足够的建筑许可证，请购买更多来建造新的建筑物",
-    MovingAverage: "平均移动",
-    MovingAverageDesc: "显示资源变化的移动平均值，而不是实时变化",
+    MovingAverage: "平均变动",
+    MovingAverageDesc: "显示资源变化的平均值，而不是实时变化量",
```

Comment:
moving average好像是个专有名词（[百度百科](https://baike.baidu.com/item/滑动平均/2640979)|[Wikipedia](https://zh.wikipedia.org/wiki/%E7%A7%BB%E5%8B%95%E5%B9%B3%E5%9D%87)），考虑到原文是"Show moving average of change"，建议如下：
Moving average seems to be a term. Considering that the original text is "Show moving average of change", the suggestions are as follows:
```suggestion
    MovingAverage: "移动平均",
    MovingAverageDesc: "显示资源变化的移动平均值，而不是实时变化量",
```